src/Symfony/Component/Validator/Tests/Constraints/DivisibleByValidatorTest.php

```diff
@@ -114,6 +114,6 @@ public function throwsOnNonNumericValuesProvider()
 
     public function provideComparisonsToNullValueAtPropertyPath()
     {
-        $this->markTestSkipped('DivisibleByValidator rejects null values.');
+       self::markTestSkipped('DivisibleByValidator rejects null values.');
     }
```

Comment:
Here the dataProvider should be static, so we need to use `self::markTestSkipped()`

Reply:
Because it unneeded, I'd prefer not merging this PR and doing the needed changes like this one in #49242